File: src/main/java/com/github/simiacryptus/aicoder/openai/ApiError.java
package com.github.simiacryptus.aicoder.openai;

public class ApiError {
    @SuppressWarnings("unused")
    public String message;
    @SuppressWarnings("unused")
    public String type;
    @SuppressWarnings("unused")
    public String param;
    @SuppressWarnings("unused")
    public Double code;

    @SuppressWarnings("unused")
    public ApiError() {
    }

    @SuppressWarnings("unused")
    public ApiError(String message, String type, String param, Double code) {
        this.message = message;
        this.type = type;
        this.param = param;
        this.code = code;
    }

}

File: src/main/kotlin/com/github/simiacryptus/openai/ApiError.kt
package com.github.simiacryptus.openai

class ApiError {
    @Suppress("unused")
    var message: String? = null

    @Suppress("unused")
    var type: String? = null

    @Suppress("unused")
    var param: String? = null

    @Suppress("unused")
    var code: Double? = null

    @Suppress("unused")
    constructor()

    @Suppress("unused")
    constructor(message: String?, type: String?, param: String?, code: Double?) {
        this.message = message
        this.type = type
        this.param = param
        this.code = code
    }
}